### ISSUE TITLE
val_loss is too high while training loss is small in intent extraction

### ISSUE THREAD

**Issue by ygk09** [NervanaSystems/nlp-architect#23]
#### Target objective:
When I am using the file [train_mlt_model.py] for intent extraction, the val_loss is very high wheras the training loss is relatively small. However, the conll eval results are all very good.
#### Steps to objective:
- Download the Dataset [nlu-benchmark] and run the file [train_mlt_model.py] with command:
   python train_mlt_model.py --dataset_path nlu-benchmark/2017-06-custom-intent-engines -b 100 -e 5
- After 5 epochs, the console shows:
13784/13784 [==============================] - 11s 817us/step - loss: 3.1504 - intent_classifier_output_loss: 0.0971 - intent_slot_crf_loss: 3.0533 - intent_classifier_output_categorical_accuracy: 0.9710 - intent_slot_crf_accuracy: 0.9671 - val_loss: 95.6078 - val_intent_classifier_output_loss: 0.0791 - val_intent_slot_crf_loss: 95.5287 - val_intent_classifier_output_categorical_accuracy: 0.9771 - val_intent_slot_crf_accuracy: 0.9740
- Here, the val_loss is 95.6 whears the training loss is only 3.15.
